docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | Entity name         | Entity description                                                                                                                                                                                                    | Entity fields                                                                                                                                                                                                       |
 +=====================+=======================================================================================================================================================================================================================+=====================================================================================================================================================================================================================+
-| CarBox              | Box which contains car box data, which could be stored and operated in Sidechain                                                                                                                                      | boxData -- contains  car box data                                                                                                                                                                                   |
+| CarBox              | Box which contains car box data, which could be stored and operated in the Sidechain. ("...which is stored and managed..."?)                                                                                                                                     | boxData -- contains  car box data                                                                                                                                                                                   |
```

Comment:
@zennermac 

| CarBox              | Is a box that contains CarBox data, which could be stored and operated in the sidechain.                                                                                                                                     | boxData -- contains  car box data                                                                                                                                                                                   |